lib/GNUmakefile

```diff
@@ -22,6 +22,9 @@
 # Include makefile which builds libgenn
 include $(GENN_PATH)/lib/GNUMakefileLibGeNN
 
+# Include makefile which builds python wrapper
+include $(GENN_PATH)/lib/GNUMakefilePyGeNN
```

Comment:
I think it would be nicer if GNUMakefilePyNN **included** GNUMakefileLibGeNN rather than the other way around

Reply:
GNUMakefilePyNN implicitly includes GNUMakefileLibGeNN by coming after it. However many variables are not very helpful because they are libgenn specific. For example, `LINK_FLAGS` contain `-lgenn` which is not needed and not built if the target is `pygenn`.

Reply:
I know, but the point of ``GNUMakefile`` to to build generateAll (which admittedly may not be clear from it's current name...) and generateAll links to libgenn so its makefile includes ``GNUMakefileLibGeNN``. Likewise the point of ``GNUMakefilePyGeNN`` is to build PyGeNN so it should include ``GNUMakefileLibGeNN`` rather than be part of ``GNUMakefile``

Reply:
How come pygenn doesn't link against libgenn?